src/main/java/at/hannibal2/skyhanni/features/event/UniqueGiftingOpportunitiesFeatures.kt

```diff
@@ -37,8 +47,7 @@ object UniqueGiftingOpportunitiesFeatures {
     private val config get() = SkyHanniMod.feature.event.winter.giftingOpportunities
 
     private fun isEnabled() = LorenzUtils.inSkyBlock && config.enabled && WinterAPI.isDecember() &&
-        (InventoryUtils.itemInHandId.endsWith("_GIFT")
-            || !config.highlighWithGiftOnly)
+        (giftNamePattern.matches(InventoryUtils.itemInHandId.asString()) || !config.highlighWithGiftOnly)
```

Comment:
Please only check the item in hand this way once per tick, not once per frame per entity alive